powerline/bindings/i3/powerline-bar.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import sys
+import time
+import i3
+from threading import Lock
+from powerline import Powerline
+from powerline.lib.monotonic import monotonic
+
+if __name__ == '__main__':
+	name = 'wm'
+	if len(sys.argv) > 1:
+		name = sys.argv[1]
```

Comment:
I guess you just copied it from `powerline-i3.py`. Do not edit, if I get rid of this I will do this for both files at once.